redisson/src/main/java/org/redisson/api/RRateLimiter.java

```diff
@@ -36,6 +36,18 @@
      *         otherwise
      */
     boolean trySetRate(RateType mode, long rate, long rateInterval, RateIntervalUnit rateIntervalUnit);
+
+    /**
+     * Updates RateLimiter's state and stores config to Redis server.
+     *
+     * @param mode - rate mode
+     * @param rate - rate
+     * @param rateInterval - rate time interval
+     * @param rateIntervalUnit - rate time interval unit
+     * @return {@code true} if rate was set and {@code false}
```

Comment:
return type should be `void` since settings set unconditionally.

Reply:
addressed